makeMLDoc.sh

```diff
@@ -14,19 +14,19 @@ ecldoc --iroot ~/source/PBblas --oroot ~/docs/PBblas --eclcc=" -I ~/source" --ex
 ecldoc --iroot ~/source/LearningTrees --oroot ~/docs/LearningTrees --eclcc=" -I ~/source" --exdocpaths ~/docs/ML_Core/xml --format "html,pdf" --exclude "internal/*.ecl,test/*.ecl,test/datasets/*.ecl" --hideInternal
```

Comment:
Please create the required folders at the very beginning of the file such as source folder, docs folder and so on.